canonical/juju-dashboard
# Inclusive naming

## Done

- Replace key non-inclusive language where possible.
- **NOTE**: Occurrences of master and slave were found in 'complete-redux-store-dump' that have not been changed.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check language changes don't break code
- Check language changes make lexical sense

## Details

Fixes: https://github.com/canonical-web-and-design/web-squad/issues/4529
